control/forms.py

```diff
@@ -110,3 +110,20 @@ class WeightConverter(forms.Form):
     unit = forms.ChoiceField(label='Unit', choices=WEIGHT_CHOICES)
     unit_to = forms.ChoiceField(label='Unit to', choices=WEIGHT_CHOICES)
     unit_value = forms.FloatField(initial=0)
+
+
+SPEED_CHOICES = [
+    ('', 'Select..'),
+    ('miles per hour', 'Miles per hour'),
+    ('foot per second', 'Foot per second'),
+    ('metre per second', 'Metre per second'),
+    ('kilometre per hour', 'Kilometre per hour'),
```

Comment:
The values usually don't have spaces: `kilometre_per_hour`

Reply:
got it, will change for all of them. Thanks